docker-compose.yml

```diff
@@ -18,8 +18,32 @@ services:
       POSTGRES_USER: perma
     volumes:
       - postgres_data:/var/lib/postgresql/data:delegated
+    ports:
+      - "127.0.0.1:54320:5432"
```

Comment:
Asking just for my own information: the database doesn't need to be exposed to the host for superset to work, right? This is just for developer convenience, if you want to use psql on your host, or similar?

Reply:
When I set it up first, I needed to add this as I was getting errors that superset wasn't able to find the db in port 5432. Having said that, I have changed a bunch of things since initial set up, so I will test it again with that removed and I will also test the network suggestion and will report here.

Reply:
It works, I pushed both updates and am looking at your other comments.